redaxo/src/core/lib/be/controller.php

```diff
@@ -170,6 +170,10 @@ public static function appendLoggedInPages()
                 ->setHasLayout(false)
                 ->setPath(rex_path::core('pages/system.phpinfo.php'))
             );
+
+        if (is_readable(ini_get('error_log'))) {
```

Comment:
warum `is_readable` ?

Reply:
Funktioniert so nicht, da `is_readable` immer false liefert.
`ini_get('error_log') !== FALSE` würde funktionieren, da wenn `error_log` nicht gesetzt ist, `ini_get` => `false` liefert.

Reply:
In der php.ini wied für error_log ein pfad gepflegt

Reply:
ah, sorry, zu kompliziert gedacht. Jetzt wird mir das auch mit is_readable klar. Dann bitte diese Kommentare ignorieren.